classes/Http/Form/Form.php

```diff
@@ -27,13 +27,10 @@
      * @param \HTMLPurifier $purifier
      * @param $options
      */
-    public function __construct($data, \HTMLPurifier $purifier, array $options = [])
+    public function __construct(array $data, \HTMLPurifier $purifier, array $options = [])
     {
-        $this->purifier    = $purifier;
-        $this->options     = $options;
-        $this->messages    = [];
-        $this->cleanData   = [];
-        $this->taintedData = [];
```

Comment:
No need to assign values in the constructor when we can assign default values to these fields.